src/main/java/com/troubleshooting/controller/AuthenticationTroubleshootController.java

```diff
@@ -0,0 +1,63 @@
+package com.troubleshooting.controller;
```

Comment:
Namespace is java reads the company domain name in reverse, so it should be `com.cyberak.troubleshooing.controller`